auto-spacing.txt

```diff
@@ -686,9 +701,14 @@ FD50..FD8F     ; N  # N   ARABIC LIGATURE TEH WITH JEEM WITH MEEM INITIAL FORM..
 FD92..FDC7     ; N  # N   ARABIC LIGATURE MEEM WITH JEEM WITH KHAH INITIAL FORM..ARABIC LIGATURE NOON WITH JEEM WITH YEH FINAL FORM
 FDF0..FDFB     ; N  # N   ARABIC LIGATURE SALLA USED AS KORANIC STOP SIGN ISOLATED FORM..ARABIC LIGATURE JALLAJALALOUHOU
 FE00..FE0F     ; N  # A   VARIATION SELECTOR-1..VARIATION SELECTOR-16
-FE10..FE19     ; O  # W   PRESENTATION FORM FOR VERTICAL COMMA..PRESENTATION FORM FOR VERTICAL HORIZONTAL ELLIPSIS
+FE10..FE16     ; N  # W   PRESENTATION FORM FOR VERTICAL COMMA..PRESENTATION FORM FOR VERTICAL QUESTION MARK
```

Comment:
@xfq This one looks like an oversight, no? Should we exclude EAW=W?

Reply:
I think we should, yes.